task/testmap.py

```diff
@@ -154,6 +154,11 @@
             'centos-8-stream',
         ]
     },
+    'cockpit-project/cockpituous': {
+        '_manual': [
+            'host',
```

Comment:
Hmm, don't we need some `@cockpit-project/bots` project selector here?

Reply:
Ah no, tests-scan pieces that together by itself, doesn't it.

Reply:
Depends what you are trying to do.
This means: "You can trigger 'host' context for cockpit-project/cockpituous repository".
That means you can trigger 'host' in cockpit-project/cockpituous or for example in bots PR you can do `host@cockpit-project/cockpituous`. This does not mean you can trigger `host` for bots from cockpitious. You can already do that just by triggering `host@cockpit-project/bots` because `host` is defined for `cockpit-project/bots`.

Reply:
oh i just named it host for no reason, I wasn't intending to run that test there atm but we definitely could do it that way, it would probably make sense to do so.

Reply:
Right, ignore me -- I originally thought you were going to trigger the bots tests, but triggering cockpituous' tests makes sense. We already run its unit tests in a workflow after all, and the tasks containers should be able to run them as well.